ee/query-service/license/manager.go

```diff
@@ -235,6 +235,25 @@ func (lm *Manager) GetLicensesV3(ctx context.Context) (response []*model.License
 	return response, nil
 }
 
+func (lm *Manager) InsertLicensesV3(ctx context.Context) *model.ApiError {
+	// get the active license from the license manager
+	activeLicenseKey := lm.activeLicense.Key
```

Comment:
no since we are inserting in the licenses_v3 table it won't be present in `activeLicenseV3.key` that will be the case once the v3 license is activated.